utility/unit_of_work/gov.go

```diff
@@ -3,12 +3,108 @@ package unit_of_work
 import (
 	"math/big"
 
+	"github.com/pokt-network/pocket/persistence"
 	coreTypes "github.com/pokt-network/pocket/shared/core/types"
 	"github.com/pokt-network/pocket/shared/utils"
 	typesUtil "github.com/pokt-network/pocket/utility/types"
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+func init() {
+	GovParams = generateParamConfigs()
+}
+
+var (
+	GovParams map[string]int
+)
+
+const (
+	BIGINT int = iota
+	INT
+	INT64
+	BYTES
+	STRING
+)
+
+func generateParamConfigs() map[string]int {
```

Comment:
I know this is overly explicit, but I feel like do you think this helps? `prepareGovParamParamTypesMap`